apps/web/src/lib/job-input/form-schema.ts

```diff
@@ -416,37 +416,43 @@ const makeZodSchemaFromJobInputFileSchema = (
 ) => {
   const { name, validations } = jobInputFileSchema;
   let maxSize = 0;
+  let minFiles = 1;
+  let maxFiles = 1;
 
   const defaultSchema = z.array(z.instanceof(File));
 
-  const schema = validations?.reduce((acc, cur) => {
-    const { validation, value } = cur;
+  validations?.forEach(({ validation, value }) => {
     switch (validation) {
       case InputValidation.MIN:
-        return acc.min(Number(value), {
-          error: t?.("Number.min", { name, value }),
-        });
+        minFiles = Number(value);
+        break;
       case InputValidation.MAX:
-        return acc.max(Number(value), {
-          error: t?.("Number.max", { name, value }),
-        });
+        maxFiles = Number(value);
+        break;
       case InputValidation.MAX_SIZE:
         maxSize = Number(value);
-        return acc;
-      case InputValidation.ACCEPT:
-        return acc;
+        break;
```

Comment:
There is not MAX_SIZE validation anymore. Shouldn't we remove the `MAX_SIZE`  InputValidation?